ajax/libs/kurento-client/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "kurento-client",
+  "npmName": "kurento-client",
+  "npmFileMap": [
+    {
+      "basePath": "/src/main/resources/META-INF/resources/js/",
+      "files": [
+        "kurento*"
+      ]
+    }
+  ],
+  "filename": "kurento-client.min.js",
+  "version": "6.0.0",
+  "description": "Media API for the Kurento Web SDK",
+  "homepage": "http://www.kurento.org",
+  "keywords": [
+    "API",
+    "kurento",
+    "WebRTC"
+  ],
+  "maintainers": [
+    {
+      "name": "kurento-maintainer-team",
+      "email": "info@kurento.org"
+    }
+  ],
+  "author": "Kurento <info@kurento.com>",
+  "bugs": {
+    "email": "kurento@googlegroups.com"
+  },
+  "repositories": [
+    {
+      "type": "git",
+      "url": "https://github.com/Kurento/kurento-client-js"
+    }
+  ],
+  "license": "LGPL-2.1"
```

Comment:
The latest version changed license to `Apache-2.0`. Ref: https://github.com/Kurento/kurento-client-js/blob/master/LICENSE